cmd/livepeer/starter/starter.go

```diff
@@ -1494,6 +1524,73 @@ func parseEthKeystorePath(ethKeystorePath string) (keystorePath, error) {
 	return keystore, nil
 }
 
+func parsePricePerUnit(pricePerUnitStr string) (*big.Rat, string, error) {
+	match := pricePerUnitRex.FindStringSubmatch(pricePerUnitStr)
+	if match == nil {
+		return nil, "", fmt.Errorf("price must be in the format of <price><currency>, provided %v", pricePerUnitStr)
+	}
+	price, currency := match[1], match[3]
+
+	pricePerUnit, ok := new(big.Rat).SetString(price)
+	if !ok {
+		return nil, "", fmt.Errorf("price must be a valid number, provided %v", match[1])
+	} else if pricePerUnit.Sign() < 0 {
```

Comment:
Why do you check it both here and in `starter.go`?

Reply:
Oh good catch. I had implemented it here for the broadcaster then realized the orchestrator check was slightly different, but didn't remove it here.